scripts/bb-test-bootable-image.sh

```diff
@@ -0,0 +1,65 @@
+#!/bin/bash
+
+# Check for a local cached configuration.
+if test -f /etc/buildslave; then
+    . /etc/buildslave
+fi
+
+# Custom test options will be saved in the tests directory.
+if test -f "../TEST"; then
+    . ../TEST
+fi
+
+TEST_BOOTABLE_IMAGE_SKIP=${TEST_BOOTABLE_IMAGE_SKIP:-"No"}
+if echo "$TEST_BOOTABLE_IMAGE_SKIP" | grep -Eiq "^yes$|^on$|^true$|^1$"; then
+    echo "Skipping disabled test"
+    exit 3
+fi
+
+ZPOOL=${ZPOOL:-"zpool"}
+ZFS=${ZFS:-"zfs"}
+CONSOLE_LOG="$PWD/console.log"
+OUTPUT_LOG="$PWD/output.log"
+
+case "$BB_NAME" in
+Fedora*)
+    true
+    ;;
+
+*)
+    echo "$BB_NAME unknown platform, skipping image test" >>$OUTPUT_LOG 2>&1
+    exit 18
```

Comment:
It's not obvious but the exact return values [matter to buildbot](http://docs.buildbot.net/latest/developer/results.html).  It controls how the output is formatted in the [waterfall](http://build.zfsonlinux.org/waterfall) and the overall build result.  This should return a 3, briefly:

  0 - success
  1 - warning
  2 - failure
  3 - skipped